From 69a34fea051d596d681ae9b25b611135bdd2dc2e Mon Sep 17 00:00:00 2001
From: hienhuynh <hien.huynh.px@rvc.renenas.com>
Date: Mon, 9 Apr 2018 11:25:40 +0700
Subject: [PATCH 487/628] ARM: DTS: r8a7742-iwg21m: Sound DMA support via SRC
 and DVC

DMA transfer to/from SRC and DVC

	DMA	DMApp
[MEM] -> [SRC] -> [DVC] -> [SSI] -> [CODEC]

	DMA     DMApp
[CODEC] -> [SSI] -> [DVC] -> [SRC] -> [MEM]

Current sound driver is supporting:
SSI3 -> DVC1 -> SRC3
SSI4 <- DVC0 <- SRC4

Signed-off-by: hienhuynh <hien.huynh.px@rvc.renenas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a7742-iwg21m.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21m.dts b/arch/arm/boot/dts/r8a7742-iwg21m.dts
index e92783c..9e86fff1 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21m.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21m.dts
@@ -517,8 +517,8 @@
 
 	rcar_sound,dai {
 		dai0 {
-			playback = <&ssi4>;
-			capture = <&ssi3>;
+			playback = <&ssi4 &src4 &dvc1>;
+			capture = <&ssi3 &src3 &dvc0>;
 		};
 	};
 };
-- 
2.7.4

